src/components/Spawn/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { useAtomValue } from 'jotai/index';
+import React, { FC, useEffect } from 'react';
+import { Group } from 'three';
+import { SpawnEffect } from './SpawnEffect.component';
+import { SpawnAnimation } from './SpawnAnimation.component';
+import { triggerCallback } from '../../services';
+import { spawnState } from '../Avatar/Avatar.component';
+
+interface SpawnProps {
+  avatar: Group;
+  onSpawnFinish?: () => void;
+}
+export const Spawn: FC<SpawnProps> = ({ avatar, onSpawnFinish }) => {
+  const animationProps = useAtomValue(spawnState);
+
+  const usesMountEffect = Boolean(animationProps?.onMountEffect?.src);
+  const usesMountAnimation = Boolean(animationProps?.onMountAnimation?.src);
```

Comment:
I think this is the best place to check for valid input source with `isValidGlbFormat()`